src/main/scala/elevate/rise/strategies/predicate.scala

```diff
@@ -57,6 +58,22 @@ object predicate {
     override def toString = "isMap"
   }
 
+  case object isMakeArray extends Strategy[Rise] {
+    def apply(e: Rise): RewriteResult[Rise] = e match {
+      case primitives.MakeArray(_) => Success(e)
+      case _ => Failure(isMakeArray)
+    }
+    override def toString = "isMakeArray"
+  }
+
+  case object isSlideSeq extends Strategy[Rise] {
```

Comment:
maybe it makes more sense to use the generic `isEqualTo` instead of specific rules like this.